src/persist/zcl_abapgit_persist_tag_utils.clas.abap

```diff
@@ -0,0 +1,94 @@
+CLASS zcl_abapgit_persist_tag_utils DEFINITION
+  PUBLIC
+  FINAL
+  CREATE PUBLIC .
+
+  PUBLIC SECTION.
+
+    CONSTANTS c_allowed_chars TYPE string VALUE `-_.a-zA-Z0-9` ##NO_TEXT.
+    " it is easier to whilelist chars, though potentially other chars can be added later if needed
```

Comment:
`whilelist` -> `whitelist` -> `allow list` -> `... allow chars, ...` ?

Reply:
yeah, i'll fix the typos ... thanks